strr-examiner-web/app/components/Host/Expansion/FilingHistory.vue

```diff
@@ -0,0 +1,106 @@
+<script setup lang="ts">
+
+defineEmits<{ close: [void] }>()
+
+const { getApplicationFilingHistory, getRegistrationFilingHistory } = useExaminerStore()
+const { isApplication, activeRecord } = storeToRefs(useExaminerStore())
+
+// business requirement: don’t show auto approval logic events - it’s implied that it’s done by the system
+const HIDDEN_EVENTS: FilingHistoryEventName[] = [
+  FilingHistoryEventName.AUTO_APPROVAL_FULL_REVIEW,
+  FilingHistoryEventName.AUTO_APPROVAL_PROVISIONAL,
+  FilingHistoryEventName.AUTO_APPROVAL_APPROVED
+]
+
+const { data: filingHistory, status } = await useLazyAsyncData<FilingHistoryEvent[]>(
+  'application-filing-history',
+  async () => {
+    let allFilingHistroy: FilingHistoryEvent[] = []
+
+    if (isApplication.value) {
+      allFilingHistroy =
+        await getApplicationFilingHistory((activeRecord.value as HousApplicationResponse).header.applicationNumber)
```

Comment:
Shouldn't this be pushed to allFilingHistroy?

Reply:
It's on new line because it would exceed max line length (eslint warning), if that's what you mean.